.github/workflows/build.yml

```
@@ -40,6 +40,50 @@ jobs:
     if: fromJSON(needs.check_source.outputs.run-docs)
     uses: ./.github/workflows/reusable-docs.yml
 
+  check_autoconf_regen:
+    name: 'Check if Autoconf files are up to date'
+    # Don't use ubuntu-latest but a specific version to make the job
+    # reproducible: to get the same tools versions (autoconf, aclocal, ...)
+    runs-on: ubuntu-24.04
+    container:
+      image: ghcr.io/python/autoconf:2024.10.11.11293396815
+    timeout-minutes: 60
+    needs: check_source
+    if: needs.check_source.outputs.run_tests == 'true'
+    steps:
+      - name: Install git
```

Is Git not already part of the image? Should it be? 

If it needs adding, can be done in a followup. 

---

Git nit:


```suggestion
      - name: Install Git
```

---

In the future image I will add it. Calver makes the some trouble about old OS image :) But based on new image or our unified image will solve this issue